src/scvmm/azext_scvmm/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67",
```

Comment:
the core version '2.0.67' is too old. Could you use '2.15.0' or above?

Reply:
Have changed it to 2.15.0